README.md

```diff
@@ -42,7 +43,11 @@ Uses the following enviroment variables:
 EXPIRE (int)
 BATCH_SIZE (int)
 DELETE_KEYS (str == 'True' or 'False')
+INDIVIDUAL_FILES (str == 'True' or 'False')
 ```
+INDIVIDUAL_FILES is a flag for whether or not you want each key to (and its values) to an individual file
```

Comment:
Nice, love that you document as you go.  Super helpful.  Might be good to stash the file name that would be output here too

Reply:
I'll be sure to add that when I update this next!